src/nupic/engine/Network.cpp

```diff
@@ -606,6 +606,29 @@ Network::getRegions() const
   return regions_;
 }
 
+
+const Collection<Link *>
+Network::getLinks() const
```

Comment:
Is this method valuable to have for use in the C++ API? If it's only for nupic's benefit, then engine_internal.i would be a better place for it.